config/locales/en.yml

```
@@ -442,6 +442,9 @@ en:
           submit_button_next_step_label: 'Save and continue with payment details'
       applications:
         index:
+          description: |
+            Applications are a representation of an API consumer. They include credentials obtained through the API's authentication method,
+            along with usage metrics and other metadata. The application plan sets the limits for each application.
```

Just an idea... 

```suggestion
            Applications represent pieces of software consuming the API. They contain a set of credentials used to access the API,
            along with usage metrics and other metadata. The application plan sets the limits for each application.
```

Also, my suggestion was actually to include this to the translations file once :grimacing: 
that's probably more cumbersome when including on the template, but I think it's also cumbersome to have duplicated text here.

---

"a representation of an API consumer" for smart and cool people, "represent pieces of software consuming the API" for people like me